ci/travis-after-success.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+curl -s https://codecov.io/bash | bash
```

Comment:
Could you change this to run it using `bash -x` instead to see what's the exact query sent at the end. I would really like to avoid running thousands of lines of code just to do a `curl -F report URL` at the end...